content/python/concepts/functions/terms/arguments/arguments.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: 'Arguments'
+Description: 'Supplies data to a funciton when it is called in a program by order of appearance.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Unless specified otherwise, arguments passed into a called function are assigned to each parameter in the order in which they appear in the function definition. Thus, they are also known as "positional arguments".
+
+## Keyword Arguments
+
+Python also supports keyword arguments — prefixing arguments with the names of parameters to assign them directly, regardless of the order.
+
+```py
+def write_a_book(name, color, clothing_item):
+  print(name + " was wearing a " + color +
+        " " + clothing_item + ".")
+
+write_a_book(color="yellow", clothing_item="raincoat", name="Jonas")
+```
+
+The output will look like this:
+
+```shell
+Jonas was wearing a yellow raincoat.
+```
+
+Keyword arguments must be passed after positional arguments.
+
+```py
+write_a_book(name="Jonas", "yellow", "raincoat")
+```
+
+The call above raises the following exception:
+
+```error
+SyntaxError: positional argument follows keyword argument
+```
+
+## Varying Arguments
+
+When defining a function, it may not be necessary to know in advance how many arguments will be needed. In such cases, a special parameter `*args` is passed in. The asterisk, known in this context as the "packing operator", packs the arguments into a [tuple](https://www.codecademy.com/resources/docs/python/tuples) stored in `args`. This tuple can then be iterated through within the function.
```

Comment:
```suggestion
When defining a function, it may not be known in advance how many arguments will be needed. In such cases, a special parameter `*args` is passed in. The asterisk, known in this context as the "packing operator", packs the arguments into a [tuple](https://www.codecademy.com/resources/docs/python/tuples) stored in `args`. This tuple can then be iterated through within the function.
```